.github/workflows/tests.yml

```diff
@@ -576,20 +587,19 @@ jobs:
         run: |
           curl -X POST -H "Content-Type: application/json" -H "${SECRET_HEADER}" -H "x-iasql-drop: true" -d "[${ACCOUNT_INDEX}]" ${LAMBDA_FUNCTION}
 
-  notify-install-all-modules-time:
+  log-image-size:
```

Comment:
lets rename the jobs to something more generic since it is not only the image size